docs/en/tutorials/getting-started.rst

```diff
@@ -717,8 +717,8 @@ the bi-directional reference:
     {
         // ... (previous code)
 
-        private $reportedBugs = null;
-        private $assignedBugs = null;
+        protected $reportedBugs = null;
```

Comment:
I'm not trying to advocate one over the other (though if I were, I'd go for private). I'm just trying to remove any confusion caused by modifiers changing half way through introductory documentation without any explanation.